.gitlab-ci.yml

```diff
@@ -87,7 +89,13 @@ setup_fusillade:
     -   source environment.$CI_COMMIT_REF_NAME
     - fi
     - cat ./roles.json | sed "s/\${stage}/${DSS_DEPLOYMENT_STAGE}/g" > temp-roles.json
-    - dcp-fusillade/scripts/setup_fusillade.py --file temp-roles.json --force testing
+    - if [ "$DSS_DEPLOYMENT_STAGE" == "dev" ]; then
+    -   FUS_STAGE='testing'
+    - else
+    -   FUS_STAGE=$DSS_DEPLOYMENT_STAGE
+    - fi
+    - echo $FUS_STAGE
+    - dcp-fusillade/scripts/setup_fusillade.py --file temp-roles.json --force $FUS_STAGE
```

Comment:
This block looks like it could be handled in a separate PR since it's unrelated to build timestamps.

Reply:
Would you prefer a change of description, or a separate PR? @xbrianh 
I suppose the overall PR is to provide the operator more context around the gitlab job that is running, rather than the current Title -- `Timestamps`

Reply:
Ah i see what you mean, this was from a different PR that was merged into master already, correct this by re-basing.